Library/Homebrew/shims/scm/git

```diff
@@ -51,14 +55,15 @@ safe_exec() {
   then
     return
   fi
+
   # prevent fork-bombs
   if [[ "$(lowercase "$arg0")" = "$SCM_FILE" || "$(realpath "$arg0")" = "$SCM_REAL" ]]
   then
     return
   fi
   if [[ "$HOMEBREW" = "print-path" ]]
   then
-    echo "$arg0"
+    echo "$(abspath "$arg0")"
```

Comment:
You can simplify this to: `abspath "$arg0"`. (No need to catch the output and then echo it again.)